src/components/Navbar/Menu.jsx

```diff
@@ -0,0 +1,34 @@
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+import clsx from "clsx";
+import styles from "./navbar.module.scss";
+import { Logo } from "../logo/Logo";
+
+export const Menu = ({ list }) => {
+  return (
+    <div className={clsx([styles.navbar__logo_container, styles.menu])}>
```

Comment:
There should be something like two or three containers on the menu. The `left` side and the `right` side. On the left side, there should be a logo and the navigation of the basic route. On the right side, there should be login and register URLs.